src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -61,6 +80,25 @@ export default function withAnnotations<P extends object>(
             this.state = { ...defaultState, activeAnnotationId };
         }
 
+        componentDidUpdate(
+            _prevProps: WithAnnotationsProps,
```

Comment:
Is prepending with underscore a convention for ignored arguments?

Reply:
Yeah, and Eslint and TS won't complain it